www/i18n/en.json

```diff
@@ -367,6 +367,7 @@
     "welcome-to-app": "Welcome to <b>{{appName}}!</b>",
     "app-name": "NREL OpenPATH",
     "to-proceed-further": "To proceed further, please scan or paste an access code that has been provided by your program administrator through a website, email, text message, or printout.",
+    "op-acc-link": "Link to Open-Access study",
```

Comment:
You can remove this - such a link is not planned at this time.

Reply:
```suggestion
```